### ISSUE TITLE
plantuml module messing org babel plantuml code block output

### ISSUE THREAD

**Issue by newsig** [doomemacs/doomemacs#7063]
### I confirm that...

- [X] I have searched the [issue tracker](https://github.com/doomemacs/doomemacs/issues), [documentation](https://docs.doomemacs.org), [FAQ](https://docs.doomemacs.org/-/faq), [Discourse](https://discourse.doomemacs.org), and [Google](https://google.com), in case this issue has already been reported/resolved.

- [X] I have read "[How to Debug Issues](https://doomemacs.org/d/how2debug)", and will use it to provide as much information about this issue as possible.

- [X] The issue can be reproduced on the **latest** available commit of Doom.

- [X] The issue can be reproduced on a stable release of Emacs, such as 27 or 28. *(Doom does not support development builds like 29+ or any version ending in .50 or .9x)*

### Expected behavior

```
#+begin_src plantuml :results verbatim
Bob -> Alice : Hello World!
#+end_src
```
This code block should output an ascii diagram in the #+results.
```
#+RESULTS:
:      ,---.          ,-----.
:      |Bob|          |Alice|
:      `-+-'          `--+--'
:        | Hello World!  |
:        |-------------->|
:      ,-+-.          ,--+--.
:      |Bob|          |Alice|
:      `---'          `-----'
```

### Current behavior

Given this code block.
```
#+begin_src plantuml :results verbatim
Bob -> Alice : Hello World!
#+end_src
```

### When plantuml module is activated in the init.el.
the output is:
```
#+RESULTS:
: /tmp/babel-7nWUGd/plantuml-G5BPHu.png

```

### When plantuml module is deactivated in the init.el.
the output is:

```
#+RESULTS:
:      ,---.          ,-----.
:      |Bob|          |Alice|
:      `-+-'          `--+--'
:        | Hello World!  |
:        |-------------->|
:      ,-+-.          ,--+--.
:      |Bob|          |Alice|
:      `---'          `-----'

```
=> i guess plantuml module is messing with babel plantuml code block

### Steps to reproduce

1. activate plantuml module in init.el.
2. doom sync.
3. open doom.
4. open org buffer
5. execute this plantuml code block:
```
#+begin_src plantuml :results verbatim
Bob -> Alice : Hello World!
#+end_src
```

### System Information

https://pastebin.com/9U4Sra0v

**Comment by p4v4n** [comment]
Also facing the same issue.
Bypassing it for now by running `M-x plantuml-download-jar` and `(setq plantuml-default-exec-mode 'jar)`.

**Comment by newsig** [comment]
> Also facing the same issue. Bypassing it for now by running `M-x plantuml-download-jar` and `(setq plantuml-default-exec-mode 'jar)`.

nope, it's no a problem with the plantuml executable. Only deactivating the plantuml module from init.el is a solution for now..

**Comment by Samoed** [comment]
This happens because blocks without a file name are assigned a `png` file [by default](https://github.com/doomemacs/doomemacs/blob/master/modules/lang/plantuml/autoload.el#L14-L15). I tried to fix it, but I don't understand how to print plain text from babel.

**Comment by newsig** [comment]
> This happens because blocks without a file name are assigned a `png` file [by default](https://github.com/doomemacs/doomemacs/blob/master/modules/lang/plantuml/autoload.el#L14-L15). I tried to fix it, but I don't understand how to print plain text from babel.


This how babel execute plantuml: `/usr/bin/plantuml -headless -ttxt -p  < /tmp/babel-vgkPmR/plantuml-rbJD3p > /tmp/babel-vgkPmR/plantuml-zpZAzl.txt`

then copy the text from the tmp file inside the buffer.
